nbs/tutorials/07-forest_plot.ipynb

```diff
@@ -17,9 +17,9 @@
    "id": "cfdb7e31",
```

Comment:
<p>As beautiful as a dabest plot is, sometimes we want to alter its appearance, be it to change the position of the legend, use different texts in the legend or the axis labels, to add or remove lines etc, or to change the color and alpha of some particular graphical elements. Sometimes we even want to move parts of the plot, such as the bootstrapped half violins onto a different axes for a meta-analysis.</p><p> </p><p>Will this be a good introduction to add into the tutorial notebook as I copied this from PostHoc.ipynb? I will adapt these wordings maybe, to make it more smooth when introducing forest plot functionalities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCLAB/DABEST-python/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>